config/default.json

```diff
@@ -1,17 +1,17 @@
 {
-  "accounts_url": "https://husky.codechain.io/explorer/api/accounts" ,
+  "accounts_url": "http://localhost:9001/api/account" ,
```

Comment:
Please do not include the diffs of the `config/default.json`

Reply:
Fixed